.github/workflows/main.yml

```
@@ -40,7 +40,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [14.x, 15.x]
+        node-version: [14.x, 16.x]
```

We really only need to focus on the LTS and the current versions of Node.js. Other versions aren't terribly important for our purposes:

```suggestion
        node-version: [16.x, 18.x]
```